.pipelines/.vsts-vhd-builder.yaml

```diff
@@ -116,7 +116,6 @@ phases:
         displayName: Copying resource to Classic Storage Account
         condition: and(eq(variables.DRY_RUN, 'False'), eq(variables['MODE'], 'default'))
       - bash: |
-          echo MODE=$(MODE) && \
```

Comment:
why do we remove this?

Reply:
not necessary here, since we already have it.

Reply:
where is it?

Reply:
Line 39:
echo "##vso[task.setvariable variable=MODE]$m"
I've also tested this by running a VHD build and observing whether the MODE variable is being propogated

Reply:
this is a logging at the beginning of each task. Let's keep all tasks consistent.